### PR TITLE
src: updated Quarkus template

### PR DESCRIPTION
**Pull request by matejvasek**
Function in template now accepts beans instead of java.lang.Object

Signed-off-by: Matej Vasek <mvasek@redhat.com>

### CODE REVIEW

File: templates/quarkus/events/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Welcome to your new Quarkus function project!
 
-This sample project contains single function: `functions.Function.echo()`,
+This sample project contains single function: `functions.Function.function()`,
```
Comment (lance) [inline templates/quarkus/events/README.md:5]:
```suggestion
This sample project contains a single function: `functions.Function.function()`,
```

Comment (matejvasek) [inline templates/quarkus/events/README.md:5]: `functions.Function.function()` this looks odd, but I don't know what would be better.

File: templates/quarkus/http/pom.xml [hunk 2]
```diff
@@ -32,7 +32,7 @@
   <dependencies>
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-funqy-http</artifactId>
+      <artifactId>quarkus-funqy-knative-events</artifactId>
```
Comment (lance) [inline templates/quarkus/http/pom.xml:35]: Why `knative-events` for HTTP function?

File: templates/quarkus/events/README.md (hunk 1 above)
Comment (lance) [reply templates/quarkus/events/README.md:5]: Possibly `functions.Function.fn()` or `functions.Function.invoke()`?

File: templates/quarkus/http/pom.xml (hunk 2 above)
Comment (matejvasek) [reply templates/quarkus/http/pom.xml:35]: It serves root uri "/".

Comment (matejvasek) [reply templates/quarkus/http/pom.xml:35]: `knative-events` also supports HTTP: GET+uriparams and POST+jsonBody.

**Review by zroubalik — APPROVED** [review]

**Pull request merged**